chat-plugins/info.js

```diff
@@ -1065,7 +1065,13 @@ var commands = exports.commands = {
 	weakness: function (target, room, user) {
 		if (!target) return this.parse('/help weakness');
 		if (!this.canBroadcast()) return;
-		var targets = target.split(/[ ,\/]/);
```

Comment:
I got `\w` and `\s` mixed up again. :( `w` just looks too much like it'd be "whitespace"

`/\s*[,\/ ]\s*/`